Services/Certificate/classes/Template/Action/Clone/class.ilCertificateCloneAction.php

```diff
@@ -112,11 +112,18 @@ public function cloneCertificate(
 			$backgroundImageFile = basename($backgroundImagePath);
 			$backgroundImageThumbnail = dirname($backgroundImagePath) . '/background.jpg.thumb.jpg';
 
-			$newBackgroundImage = $newCertificate->getBackgroundImageDirectory() . $backgroundImageFile;
-			$newBackgroundImageThumbnail = str_replace($webDir, '', $newCertificate->getBackgroundImageThumbPath());
-
-			if ($this->fileSystem->has($backgroundImagePath)) {
-				if ($this->fileSystem->has($newBackgroundImage)) {
+            $newBackgroundImage = '';
+            $newBackgroundImageThumbnail = '';
+
+			if ($this->fileSystem->has($backgroundImagePath) &&
```

Comment:
Please check your tabs/spaces settings. In `release_5-4` you MUST use tabs, in `trunk` you MUST use spaces.